src/views/About.vue

```diff
@@ -19,22 +19,52 @@
               </v-list-item-content>
               <v-list-item-avatar size="100">
                 <img
-                  :src="require('@/assets/img/' + person.alias + '-avatar.jpg')"
+                  :src="require(`@/assets/img/${person.alias}-avatar.jpg`)"
                   :alt="person.alias"
                 />
               </v-list-item-avatar>
             </v-list-item>
             <v-card-actions class="white justify-center">
               <v-btn
                 icon
-                :href="'https://github.com/' + person.alias"
+                :href="`https://github.com/${person.alias}`"
                 target="_blank"
+                rel="noreferer"
               >
                 <v-icon>mdi-link</v-icon>
               </v-btn>
             </v-card-actions>
           </v-card>
         </v-col>
+        <v-col>
+          <v-card class="mx-auto" width="270" outlined>
+            <v-list-item three-line>
+              <v-list-item-content>
+                <div class="overline mb-4">社区贡献者</div>
```

Comment:
(not eye-catching enough?)

Reply:
Any suggestions? Should I use a different background color?